src_py/sprite.py

```diff
@@ -66,41 +66,37 @@
 
 """
 
-##todo
-## a group that holds only the 'n' most recent elements.
-## sort of like the GroupSingle class, but holding more
-## than one sprite
-##
-## drawing groups that can 'automatically' store the area
-## underneath so they can "clear" without needing a background
-## function. obviously a little slower than normal, but nice
-## to use in many situations. (also remember it must "clear"
-## in the reverse order that it draws :])
-##
-## the drawing groups should also be able to take a background
-## function, instead of just a background surface. the function
-## would take a surface and a rectangle on that surface to erase.
-##
-## perhaps more types of collision functions? the current two
-## should handle just about every need, but perhaps more optimized
-## specific ones that aren't quite so general but fit into common
-## specialized cases.
+# TODO:  a group that holds only the 'n' most recent elements.
+#        sort of like the GroupSingle class, but holding more
+#        than one sprite
+#
+# drawing groups that can 'automatically' store the area
+# underneath so they can "clear" without needing a background
+# function. obviously a little slower than normal, but nice
+# to use in many situations. (also remember it must "clear"
+# in the reverse order that it draws :])
+#
+# the drawing groups should also be able to take a background
+# function, instead of just a background surface. the function
+# would take a surface and a rectangle on that surface to erase.
+#
+# perhaps more types of collision functions? the current two
+# should handle just about every need, but perhaps more optimized
+# specific ones that aren't quite so general but fit into common
+# specialized cases.
 
-import pygame
-from pygame import Rect
-from pygame.time import get_ticks
 from operator import truth
 
-# Python 3 does not have the callable function, but an equivalent can be made
-# with the hasattr function.
-if 'callable' not in dir(__builtins__):
-    callable = lambda obj: hasattr(obj, '__call__')
+import pygame
+
 
 # Don't depend on pygame.mask if it's not there...
```

Comment:
At this point we can probably just assume pygame.mask is there.

Reply:
👍